CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 6.1.2
+* Fix 'select' method to allow deselecting selection
```

Comment:
Fix 'select' method in MockISelectionManager to allow deselecting selection

Reply:
Fixed.